tools/offloader/offloader.cpp

```diff
@@ -105,10 +105,12 @@ int run() {
       APIToUse = GPUAPI::DirectX;
       outs() << "Using DirectX API\n";
     } else if (*reinterpret_cast<const uint32_t *>(
-                   PipelineDesc.Shaders[0].Shader->getBuffer().data()) == 0x07230203) {
+                   PipelineDesc.Shaders[0].Shader->getBuffer().data()) ==
+               0x07230203) {
```

Comment:
Really? Might be worth a variable for `PipelineDesc.Shaders[0].Shader->getBuffer()` to clean up this whole if-else chain anyway.